packages/bitcoin-wallet-provider/lib/BitcoinWalletProvider.ts

```diff
@@ -540,7 +586,15 @@ export default class BitcoinWalletProvider
       const walletAddress = walletAddresses.find((walletAddr) =>
         addresses.find((addr) => walletAddr.address === addr),
       );
-      if (walletAddress) return walletAddress;
+
+      if (walletAddress) {
+        // Increment max addresses to derive by 100, if found.
```

Comment:
Would be good to change this comment to `Increment max addresses to derive by 100 if found within 100 addresses of maxAddressesToDerive`

Was confusing when I first read it because it seemed like it would derive every single time